task.sql

```diff
@@ -1 +1,40 @@
-# Write your SQL code for the database creation here. Good luck! 
+CREATE DATABASE ShopDB;
+USE ShopDB;
+CREATE TABLE Products (
+     ID INT AUTO_INCREMENT,
+     Name VARCHAR(50),
+     Description VARCHAR(100),
+     Price INT,
```

Comment:
Consider changing the `Price` column type to `DECIMAL` to handle fractional prices, which are common in product pricing.